daemon/logger/jsonfilelog/jsonfilelog_test.go

```diff
@@ -54,6 +54,51 @@ func TestJSONFileLogger(t *testing.T) {
 	}
 }
 
+func TestJSONFileLoggerWithTags(t *testing.T) {
+	cid := "a7317399f3f857173c6179d44823594f8294678dea9999662e5c625b5a1c7657"
+	cname := "test-container"
+	tmp, err := ioutil.TempDir("", "docker-logger-")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmp)
+	filename := filepath.Join(tmp, "container.log")
+	l, err := New(logger.Info{
+		Config: map[string]string{
+			"tag": "{{.ID}}/{{.Name}}", // first 10 characters of ContainerID and full ContainerName
```

Comment:
@jnovack - I think I screwed up by branch when trying to rebase. Is there someone I should reach out to to validate just the branch / rebase. I suspect it may just be easier to rebranch my earlier code, rebase correctly and submit a new PR, but I would like a bit of advice on it.